client/lib/analytics/cart.js

```diff
@@ -10,12 +10,16 @@ import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { recordAddToCart } from 'calypso/lib/analytics/record-add-to-cart';
 import { getAllCartItems } from 'calypso/lib/cart-values/cart-items';
 
+function difference( items1, items2 ) {
+	return items1.filter( ( a ) => ! items2.some( ( b ) => isEqual( a, b ) ) );
+}
+
 export function recordEvents( previousCart, nextCart ) {
```

Comment:
Just out of curiosity, is this function used anywhere? I can't find any use-cases.

Reply:
Indeed, only the `recordAddEvent` export is used.

Reply:
Good call 👍  Will close this one in favor of #52636 that does the removal.